api/src/main/java/org/mandulis/mts/auth/AttachmentController.java

```diff
@@ -0,0 +1,42 @@
+package org.mandulis.mts.controller;
+
+import org.mandulis.mts.dto.AttachmentDto;
+import org.mandulis.mts.exception.EmptyAttachmentException;
+import org.mandulis.mts.service.AttachmentService;
+import org.mandulis.mts.attachment.AttachmentValidationService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+@RestController
+@RequestMapping("/api/v1/user/tickets")
+class AttachmentController {
+
+    private final AttachmentValidationService attachmentValidationService;
+    private final AttachmentService attachmentService;
+
+    AttachmentController(AttachmentValidationService attachmentValidationService, AttachmentService attachmentService) {
+        this.attachmentValidationService = attachmentValidationService;
+        this.attachmentService = attachmentService;
+    }
+
+    @PostMapping("/{ticketId}/attachments")
+    ResponseEntity<Void> upload(
```

Comment:
This should return a ResponseEntity<ApiResponse<AttachmentResponse>>> 
AttachmentResponse being the response DTO